test/commonControlTests/NumberControl.spec.ts

```diff
@@ -59,6 +59,32 @@ suite('NumberControl e2e tests', () => {
                 (requestHandler.getSerializableControlStates()[TEST_CONTROL_ID] as NumberControlState).value,
             ).eq(16);
         });
+
+        test('APL event, no confirmation, no validation required', async () => {
+            const requestHandler = new ControlHandler(new NumberControlManager());
+            await testE2E(requestHandler, [
+                'U: ',
+                TestInput.of(GeneralControlIntent.of({})),
+                'A: What number?',
+                'U: ',
+                TestInput.userEvent({
```

Comment:
nit: here and elsewhere.  best to create a utility function that reduces the boilerplate.  I have one coming in Questionnaire PR so no need to do it here.. but we can come back and simplify this code later this week.